conf/supervisord-dev.conf

```diff
@@ -0,0 +1,51 @@
+[supervisord]
+nodaemon = true
+silent = true
```

Comment:
I found that when I <kbd><kbd>Ctrl</kbd>+<kbd>c</kbd></kbd> the `make dev` process supervisor prints several lines of logs _after_ my shell has drawn my next prompt, forcing me to hit <kbd>Enter</kbd> to get a clean prompt. Turning off supervisor's own logging (and letting it only echo the logs of the processes it's running) seems to fix that, and also removes some not-really-interesting noise from the output.